### PR TITLE
Remove time estimates

### PR DESCRIPTION
**Pull request by ikaur9**
Remove time estimates from units 1-14

### CODE REVIEW

File: unit_09_plan.md [hunk 1]
```diff
@@ -23,5 +20,5 @@ In this unit, you are going to learn about uncertainty estimates for these estim
 
 1. To understand that OLS regression coefficients' standard errors are a produce of sampling, and to appreciate the similarity of this concept to measures of uncertainty from hypothesis in earlier units. 
 
-2. To conduct appropriate tests within the Frequentist testing framework. These tests include tests for a null hypothsis for individual coefficients, and tests for a null hypothesis about an ensemble of coefficients. 
+2. To conduct appropriate tests within the Frequentist testing framework. These tests include tests for a null hypothesis for individual coefficients, and tests for a null hypothesis about an ensemble of coefficients.
```
Comment (d-alex-hughes) [inline unit_09_plan.md:23]: And a spell check too! Thank you @ikaur9 !

File: unit_10_part_2.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 ## Capturing Nonlinear Relationships 
 
-Although the statistical models that you are bulding are a linear combination of terms, it does not mean that the outcome values only exist on a plane. 
+Although the statistical models that you are building are a linear combination of terms, it does not mean that the outcome values only exist on a plane.
```
Comment (d-alex-hughes) [inline unit_10_part_2.md:5]: again, thanks for the spell check.

**Review by d-alex-hughes — APPROVED** [review]
Thank you so much @ikaur9! This is great. Merging onto the main branch.
